tests/requirements.in

```diff
@@ -0,0 +1,5 @@
+pytest >=3.6.4,!=3.7.0,!=3.7.1,!=3.7.2,!=3.7.3,!=3.7.4,!=3.8.0,!=3.8.1 # https://github.com/pytest-dev/pytest-bdd/issues/250
```

Comment:
This refers to an issue that was closed in `pytest-bdd`, so does the bound still apply? Given the `pytest` version in the rendered `requirements.txt` is `4.3.0`, can't this be `pytest >= 4.3.0` instead?

Reply:
well I surely didnt get the fixture issue so going for 4.3.0